src/actions/press.js

```diff
@@ -40,8 +41,8 @@ const shapesFactory = {
   },
 };
 
-export default function press(x, y, Dispatcher, Store) {
-  switch (Store.getMode()) {
+export default function press(x, y, Dispatcher, store) {
+  switch (store.getMode()) {
```

Comment:
Can we leave this as it was before? Store. So there's no conflicts with ongoing code.